MovieJournalMVC/Controller/DetailViewController.swift

```diff
@@ -11,14 +11,22 @@ import UIKit
 class DetailViewController: UIViewController {
 
     @IBOutlet weak var detailDescriptionLabel: UILabel!
-
-
+    @IBOutlet weak var movieTitleLabel: UILabel!
+    @IBOutlet weak var movieReviewLabel: UILabel!
+    
     func configureView() {
         // Update the user interface for the detail item.
         if let detail = detailItem {
-            if let label = detailDescriptionLabel {
-                label.text = detail.description
+            if let label = movieTitleLabel {
```

Comment:
There's no need to unwrap labels to set text. Optional chaining can be used to set properties. Additionally, since these outlets are implicitly unwrapped anyway, you don't need and probably don't want this check.

Reply:
As you saw, I am no longer using this class, rather, I am using EntryDetailViewController for the detail screen.